app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  after_save :update_post_counter
+
+  def update_post_counter
+    author.increment!(:posts_counter)
+  end
+
```

Comment:
Good job adding post models, however, it could be better to make the `update_post_counter` method `private` because it'll be a private method used within our class to update the posts_counter attribute of the associated author. It's called as part of the after_save callback.

we'll have something like this:

````ruby
class Post < ApplicationRecord
  has_many :comments
  belongs_to :author, class_name: 'User'
  has_many :likes
  after_save :update_post_counter

 def recent_comments
    comments.order(created_at: :desc).limit(5)
  end

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
end